ts/components/session/settings/SessionSettings.tsx

```diff
@@ -198,7 +198,7 @@ export class SettingsView extends React.Component<SettingsViewProps, State> {
   }
 
   public async validatePasswordLock() {
-    const enteredPassword = String($('#password-lock-input').val());
+    const enteredPassword = String(($('#password-lock-input') as any).val());
```

Comment:
in regard to `($('#password-lock-input') as any)`, is this done because TS is complaining?
If so might be best for us to check if we have the typings for jQuery, once we add that in it shouldn't complain

Reply:
WebdriverIO and JQuery types are clashing and thus we do this.